src/commands/provision/provision.go

```diff
@@ -0,0 +1,20 @@
+package provision
+
+import (
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var ProvisionCmd = &cobra.Command{
+	Use:   "provision",
+	Short: "Provision a resource",
+	Long:  `Provision a resource`,
+	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Help()
+		os.Exit(0)
+	},
+}
+
+func init() {
+	ProvisionCmd.AddCommand()
```

Comment:
Need to add `ProvisionServerCmd` to this command

Reply:
Added